modules/nw-route-admission-policy.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+// * networking/routes/route-configuration.adoc
+
+[id="nw-route-admission-policy_{context}"]
+= Configuring the route admission policy
+
+Cluster administrators can run applications in multiple namespaces with the same domain name by configuring the `RouteAdmissionPolicy` `ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK` environment variable.
```

Comment:
I would propose something like this (lifted from the [enhancement proposal](https://github.com/openshift/enhancements/blob/master/enhancements/ingress/openshift-route-admission-policy.md)):

>Administrators and application developers would like to be able to run applications in multiple namespaces with the same domain name. This is for organizations where multiple teams develop microservices that are exposed on the same host name.

>Allowing claims across namespaces should only be enabled for clusters with trust between namespaces, otherwise a malicious user could take over a host name. For this reason, the default admission policy disallows host name claims across namespaces.

Reply:
Thanks! I'm adding this and modifying it a bit for the product docs.